### ISSUE TITLE
The results of `ord=-1` with `linalg.norm()` and `min(sum(abs(x), dim=0))` are different

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#136412]
### 🐛 Describe the bug

[The doc](https://pytorch.org/docs/stable/generated/torch.linalg.norm.html) of `linalg.norm()` says that `ord=-1` is `min(sum(abs(x), dim=0))` as shown below:

```
ord          norm for matrices          norm for vectors
...          ...                        ...
-1           min(sum(abs(x), dim=0))    as below
```

But the results of `ord=-1` with `linalg.norm()` and `min(sum(abs(x), dim=0))` are different as shown below:

```python
import torch
from torch import linalg

my_tensor = torch.tensor([-3., -2., -1., 0., 1., 2.])

print(linalg.norm(input=my_tensor, ord=-1)) # tensor(0.)
print(torch.min(torch.sum(torch.abs(input=my_tensor), dim=0))) # tensor(9.)
```

### Versions

```python
import torch

torch.__version__ # '2.3.0'
```

**Comment by awgu** [comment]
maybe because you passed in a vector, not a matrix?

**Comment by hyperkai** [comment]
[awgu](https://github.com/awgu). Yes, you right. Thanks.
